react/week4/my-app/src/FindUser.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import context from "./context";
+export default function FindUser() {
+  const { handelUserSearch } = React.useContext(context);
+  const form = (
+    <form>
+      {" "}
+      <input
+        type="text"
+        id="userName"
+        placeholder="search for user"
+        onChange={handelUserSearch}
+      />
+    </form>
+  );
+
+  return <React.Fragment>{form}</React.Fragment>;
```

Comment:
Fragments are most useful when you need to render two or more adjacent elements and don't want the overhead of adding an extra div just to appease react. I would simply return your input here instead: 
```suggestion
  return  <input
           type="text"
        id="userName"
        placeholder="search for user"
        onChange={handelUserSearch}
      />
```